docs/.vuepress/config.ts

```diff
@@ -17,13 +17,11 @@ export default defineConfig({
     ['link', {rel: 'icon', href: '/favicon.ico'}],
   ],
   configureWebpack: (config, isServer) => {
-    configureWebpack: (config) => {
-      return {
-        plugins: [
-          new HtmlWebpackPlugin(),
-          new HtmlWebpackInlineSourcePlugin(HtmlWebpackPlugin),
-        ],
-      };
+    return {
+      plugins: [
+        new HtmlWebpackPlugin(),
+        new HtmlWebpackInlineSourcePlugin(),
+      ],
```

Comment:
@benmccann I guess you made a typo here

https://github.com/chartjs/Chart.js/commit/9258f250b6187a72cbdf3bf177d9452097149565#diff-d6bc2c4b08fb339d0b296294e24adcaf16db0294915900c58b33ddce5033385b

Reply:
@benmccann Why did you add this?

Reply:
`configureWebpack` can return a function as well, so I think what I had was valid, but this seems equivalent and simpler

Reply:
@benmccann It was a label

```
configureWebpack: (config, isServer) => {
    configureWebpack: (config) => { // error TS7028: unused label
      return {
        plugins: [
          new HtmlWebpackPlugin(),
          new HtmlWebpackInlineSourcePlugin(HtmlWebpackPlugin),
        ],
      };
    };
  },
```

Reply:
So do we need this? Currently, in master docs are built without these plugins

Reply:
Ah, yeah. Though maybe it should just be removed if it's not needed then